assets/js/modules/analytics/components/module/ModulePopularPagesWidgetGA4.js

```diff
@@ -127,7 +128,14 @@ function ModulePopularPagesWidgetGA4( props ) {
 		} )
 	);
 
-	if ( ! loaded || isGatheringData === undefined ) {
+	const isGA4ReportingTourActive = useSelect(
+		( select ) => select( CORE_USER ).getCurrentTour() === ga4ReportingTour
```

Comment:
This uses the current tour as this will automatically be `false` as soon as the tour is no longer showing. This is not the case for the other UI key which is used to invoke the tour initially.